### DISCUSSION TITLE
reTerminal DM SenseCraft Edge OS Getting Started | Seeed Studio Wiki

### DISCUSSION THREAD

**Discussion by giscus** [Seeed-Studio/wiki-documents#341] [Q&A]
# reTerminal DM SenseCraft Edge OS Getting Started | Seeed Studio Wiki

reTerminal DM SenseCraft Edge OS Getting Started

https://wiki.seeedstudio.com/reterminal-dm-sensecraft-edge-os-intro/

**Comment by anderml1955** [comment]
Where can I find the SenseCraft_Edge OS installation instructions and downloads?

**Reply by MatthewJeffson** [reply]
Hi! I will check with my colleague about that! Thank you!

**Reply by MatthewJeffson** [reply]
Hi! We are still working on the publicable version and the way to flash it. 
Is it possible that I can ask why you want to flash it back and which scenarios you want to use it?
Thanks!

**Reply by anderml1955** [reply]
I want to investigate the potential for a project. Not having a way to re-install the delivered OS without warning users they cannot revert back to it once a different OS is installed is wrong. It seems all you have to do is make the img file available for download and installation, much like all the other OS options are.

**Reply by MatthewJeffson** [reply]
I'm terribly sorry for such delay and we are indeed supposed to publish the img file at the first place.
Thank you very much for your interest in our products!

**Reply by peterpanstechland** [reply]
Sorry for the wait, where our R'n'D team has been working hard on preparing this image file for users who willing to restore the SenseCraft Edge OS, here is the current version https://sourceforge.net/projects/reterminal-dm/files/Sensecraft-edge-OS-V0.3.10.tar.gz/ . Please note that the Default HMI APP will took sometime to be loaded for the first time as the system will perform some system health check when first booted.

**Comment by trlafleur** [comment]
I have the same issue, Where can I find the SenseCraft_Edge OS for reTerminal DM installation instructions and downloads?

**Reply by MatthewJeffson** [reply]
Hi! We are still working on the publicable version and the way to flash it. 
And may I ask why you want to flash it back and which scenarios you want to use it?
Thanks!

**Comment by Anarchy05** [comment]
Hi There, I would also like a download link to restore the OS, as I want to explore using it as a pure HMI where user cannot access backend and want to run other custom written softwares. Also need to see how operation works on this and can I run my own frontend etc. Waiting for my reTerminal DM devices in post and should be ariving soon

**Reply by MatthewJeffson** [reply]
Thanks for the sharing! We will test the publicable version and the way to flash it. Will be published as soon as possible.:D

**Comment by trlafleur** [comment]
Is this the current version ??

https://sourceforge.net/projects/reterminal-dm/files/

**Reply by peterpanstechland** [reply]
yes, I have upload the current version https://sourceforge.net/projects/reterminal-dm/files/Sensecraft-edge-OS-V0.3.10.tar.gz/ , you could use rpi-imager to flash the image, but to be noted that the Sensecraft Edge OS APP will perform some system health check which will took sometime to load.

**Comment by trlafleur** [comment]
We have loaded this image and are very disappointed, ChirpStack is ver
1.0.2, NodeRed is 1.0, and Mouse action are rotated 90 deg,...  Not a good start!!

looks like it needs some serious updates....

~~ _/) ~~~~ _/) ~~~~ _/) ~~~~ _/) ~~

Tom Lafleur



---------- Forwarded message ---------
From: peterpanstechland ***@***.***>
Date: Sun, Jul 9, 2023 at 5:43 PM
Subject: Re: [Seeed-Studio/wiki-documents] reTerminal DM SenseCraft Edge OS
Getting Started | Seeed Studio Wiki (Discussion #341)
To: Seeed-Studio/wiki-documents ***@***.***>
Cc: Tom Lafleur ***@***.***>, Comment ***@***.***>


yes, I have upload the current version
https://sourceforge.net/projects/reterminal-dm/files/Sensecraft-edge-OS-V0.3.10.tar.gz/
, you could use rpi-imager to flash the image, but to be noted that the
Sensecraft Edge OS APP will perform some system health check which will
took sometime to load.

—
Reply to this email directly, view it on GitHub
<https://github.com/Seeed-Studio/wiki-documents/discussions/341#discussioncomment-6398921>,
or unsubscribe
<https://github.com/notifications/unsubscribe-auth/ABC4EKZPGUJKZI6XWX6S36DXPNFZLANCNFSM6AAAAAAZJ3WVIY>
.
You are receiving this because you commented.Message ID:
***@***.***
com>

**Reply by MatthewJeffson** [reply]
Sorry to keep you waiting and we are indeed working on a new version now.
Thank you so much for your patiences!

**Comment by trlafleur** [comment]
Matthew...
Some non-software related issues...
1) What is the correct mfg and part number for the power and I/O connectors used on the DM
2) You need to sell an IOT kit, with the correct WM1302, cable, antenna, and the proper 2mm screws
3) Same for WiFi antenna kit, and software to switch CM4 module from internal to external antenna.

thanks...

**Reply by MatthewJeffson** [reply]
Hello!
I think we do have [reTerminal DM with Optional Interfaces and Modules](https://www.seeedstudio.com/reTerminal-DM-LTE-Cat-4-EC25-Bundle-p-5675.html). And regarding mfg and part number I am not sure I can help you with, but I will try to ask around. :D

**Comment by DSI-AlexA** [comment]
The sensecraftEdgeOS comes with a "seeed" sudo user account prebaked into it. It would be very helpful to have the password to this account.

**Reply by MatthewJeffson** [reply]
Hello! May I have your email so we can give you the authority?

**Comment by amirallesaviartec** [comment]
Hello! Can you give me some guidance with this problem? When I try to install anything from apt-get I get the following error:

W: Not using locking for read only lock file /var/lib/dpkg/lock-frontend
W: Not using locking for read only lock file /var/lib/dpkg/lock

Thank you very much. The device is fantastic.

**Reply by amirallesaviartec** [reply]
Nevermind, we could manage it. Thanks!

**Reply by ridhwanw** [reply]
Hello.. tell me what you do? Please

**Comment by cgjgh** [comment]
Hello, 
Is Sensecraft Edge OS still being developed/maintained? Also is it available on GitHub and can we fork it to contribute or develop for our own purposes? 

Thank you.

**Reply by nfs0619** [reply]
Hello, thank you for your interest!

Sensecraft Edge OS is no longer being actively developed or maintained. Our current default system is based on Raspberry Pi.

If you're interested in contributing, we’d love for you to check out our contributor program:

* [Contributor Introduction](https://wiki.seeedstudio.com/contributors/)
* [Contributor Project Page](https://github.com/orgs/Seeed-Studio/projects/6/views/1?pane=issue&itemId=30957479)

We look forward to more exchanges and collaboration with you!
